Description:
 Fix a bug reported by Simon L. B. Nielsen:
 .
 If an SCTP endpoint receives an ASCONF with a wildcard
 lookup address and  incorrect verification tag, the system
 crashes.
 .
 This bug was found by Shaun Colley.
Origin: upstream, http://svnweb.dragonflybsd.org/base?view=revision&revision=239447
Bug-Debian: http://bugs.debian.org/686962

--- kdragonflybsd-9-9.0.orig/sys/netinet/sctp_pcb.c	2011-11-24 17:58:14.000000000 +0000
+++ kdragonflybsd-9-9.0/sys/netinet/sctp_pcb.c	2012-09-30 17:13:51.837771919 +0100
@@ -2073,8 +2073,6 @@
 	struct sctp_nets *net;
 	struct sctp_tcb *stcb;
 
-	*netp = NULL;
-	*inp_p = NULL;
 	SCTP_INP_INFO_RLOCK();
 	head = &SCTP_BASE_INFO(sctp_asochash)[SCTP_PCBHASH_ASOC(vtag,
 	    SCTP_BASE_INFO(hashasocmark))];
